atlp-rwanda/e-commerce-lydia-32-fn
# ft-187570820-PasswordExpirationPrompt

### What does this PR do?
This PR implements a feature that prompts users to change their passwords every specified amount of time since the last update. If the password has expired, the user will be redirected to a Password Update page and no other operation will be possible until the password is updated.
### Description of Task to be completed?

- Create a Password Update page for users to change their passwords.
- Add logic to check if the password has expired upon user login.
- Redirect users to the Password Update page if their password has expired.
- Ensure no other operations can be performed until the password is updated.

### How should this be manually tested?

- Log in to the application with expired password.
- Verify that the user is redirected to the Password Update page.
- Try to perform other operations and confirm they are blocked until the password is updated.
- Update the password and ensure the user can access other operations normally.

#### N/A

### Any background context you want to provide?

#### N/A

### Screenshots (if appropriate)

#### N/A

### Questions:

#### N/A

### What are the relevant pivotal tracker stories

#### N/A
